stories/About/Developer Guide.mdx

```diff
@@ -0,0 +1,72 @@
+import Image from "next/image";
+import '../styles.css';
+
+import supabase from '../assets/supabase_gridIron.png';
+import server from '../assets/supabase_server.png';
+import client from '../assets/client_component.png';
+
+## Developer Guide
+
+### Table of Contents
+ - [Getting Started](#getting-started)
+ - [Pushing local branch to remote branch](#pushing-local-branch-to-remote-branch)
+ - [Connecting to Supabase](#connecting-to-supabase)
+ - [Query Supabase data from Next](#query-supabase-data-from-next)
+    
+
+## Getting Started
+
+  <code>git clone --single-branch --branch develop https://github.com/LetsGetTechnical/gridiron-survivor.git</code>
+
+  Install the required packages with pnpm (NOT npm or yarn!)
+
+  <code>pnpm install</code>
+
+  Create a new branch for whatever you’re working on in the format firstname/feature-description, then check out your new branch. For example:
+
+  <code>git branch richard/user-picks-page && git checkout richard/user-picks-page</code>
+  
+  Code your feature! 🤓
+
+## Pushing local branch to remote branch
+
+  For the first push from local to remote, use --set-upstream origin along with the name of your local branch, like so:
+
+  <code>git push --set-upstream origin richard/user-picks-page</code>
+
+  Subsequent pushes will now go to your remote upstream branch.
+
+## Connecting to Supabase
+
+  In the project’s root folder, create an .env.local file:
+
+  <Image src = {supabase} alt = {""}/>
```

Comment:
@CorinaMurg Do you want these images as decorative alt tags or should we add descriptions for them?

Reply:
Follow this coding style format. It's not ideal to add spaces between `=` sign. 
```suggestion
  <Image src={ supabase } alt={ "" } />
```

Reply:
@shashilo, @choir27 
> Do you want these images as decorative alt tags or should we add descriptions for them?

The `alt` can be empty for the first image (the one showing the `env` file highlighted) because the info in the image is also available as text (“create env file at the root…”)

All the other images contain info that we want the visitor to know, so yes, we have to make sure it’s available to screen reader users as well.

Normally, when an image shows code or snapshot from documentation, the `alt` can be empty, but then we have to add a link to a codepen with the actual code or the documentation.

In this case, a better solution is to embed the code on the page with `<pre><code>` `…<code><pre>` tags and also style it to look like it’s in vscode.

Reply:
Client Component Codepen Example: 
![Screenshot 2024-04-12 144604](https://github.com/LetsGetTechnical/gridiron-survivor/assets/66279068/64b91dee-2ec9-48bb-8557-6538654346a9)
https://codepen.io/choir27/pen/zYXaVGY?editors=0010

Server Component Codepen Example:
![Screenshot 2024-04-12 144551](https://github.com/LetsGetTechnical/gridiron-survivor/assets/66279068/a3d440f2-a3b5-4ba9-8094-27ea7e935937)
https://codepen.io/choir27/pen/oNOyrjz?editors=0110

Reply:
@choir27 You did not take care of @CorinaMurg's comment about adding descriptive content to images.

Reply:
@shashilo sorry about that, I'll fix that right now!